tests/testMaskedImagePersistence1.py

```diff
@@ -106,20 +106,12 @@ def testBoostPersistence(self):
 
```

Comment:
This test needs fixing. The two testX method call self.checkImages(...), which does nothing (the body is `pass` despite the comforting documentation). I suggest changing it to call assertMaskedImagesEqual, but that will require having the test case inherit from `lsst.utils.tests.TestCase`